pkg/controller/statusmanager/status_manager.go

```diff
@@ -312,6 +312,52 @@ func (status *StatusManager) SetNotDegraded(statusLevel StatusLevel) {
 	status.setNotDegraded(statusLevel)
 }
 
+// syncProgressing syncs the current Progressing status
+func (status *StatusManager) syncProgressing() {
+	for _, c := range status.failing {
```

Comment:
I think you need to check here if the status.Type is Progressing.

Reply:
fixed